awkward/arrow.py

```diff
@@ -263,6 +264,18 @@ def popbuffers(array, tpe, buffers, length):
                 return awkwardlib.BitMaskedArray(mask, out, maskedwhen=False, lsborder=True)
             else:
                 return out
+                
+        elif isinstance(tpe, pyarrow.lib.LargeListType):
+            assert getattr(tpe, "num_buffers", 2) == 2
+            mask = buffers.pop(0)
+            offsets = awkwardlib.numpy.frombuffer(buffers.pop(0), dtype=ARROW_LARGEINDEXTYPE)[:length + 1]
+            content = popbuffers(None if array is None else array.flatten(), tpe.value_type, buffers, offsets[-1])
+            out = awkwardlib.JaggedArray.fromoffsets(offsets, content)
+            if mask is not None:
+                mask = awkwardlib.numpy.frombuffer(mask, dtype=ARROW_BITMASKTYPE)
+                return awkwardlib.BitMaskedArray(mask, out, maskedwhen=False, lsborder=True)
+            else:
+                return out
```

Comment:
I see—only for lists, for now at least.

Reply:
I'll check to see if there are others. It's a recent addition in 0.15.x.

Reply:
In the if-statement, you could add a `hasattr` so that it will work for old and new versions.